src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="form" action="#"
+      metod="post" data-qa="big">
+      <label>
+        <input
+          type="submit"
+          name="big__submit"
+          class="icon"
+        >
+      </label>
+      <label> 
+        <input
+          type="search"
+          name="big__search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="field"
+          data-qa="keypress"
+        >
+      </label> 
+    </form>
+
+    <form class="form form__small" action="#" 
+      metod="post" data-qa="small">
+      <label>
+        <input
+          type="submit"
+          name="small__submit"
+          class="icon icon__small"
+        >
+      </label>
+      <label>
+        <input
+          type="search" 
+          name="small__search" 
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="field small__field"
```

Comment:
Uncorrect class name for small field. Please fix like in checklist, _element--modifier_.